packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -311,22 +281,35 @@ test('govern offerFilter', async t => {
   // 44 tested above
   t.is(currentState.offerToUsedInvitation[46].value[0].description, 'Voter0');
 
+  // Call for a vote ////////////////////////////////
+
   /** @type {import('@agoric/smart-wallet/src/invitations').ContinuingInvitationSpec} */
-  const getVoteSpec = {
+  const proposeInvitationSpec = {
     source: 'continuing',
-    previousOffer: 46,
-    invitationMakerName: 'makeVoteInvitation',
-    invitationArgs: harden([yesPosition, questionHandle]),
+    previousOffer: 44,
```

Comment:
Help me understand why `previousOffer` is changing?

Reply:
This is the differ getting confused. This invitation is for `VoteOnPauseOffers` which needs 44, instead of `makeVoteInvitation` which needed 46.

But I've been replacing numbers with strings so I'll do that here too.